scout/server/blueprints/variants/controllers.py

```diff
@@ -833,120 +833,154 @@ def callers(variant_obj, category='snv'):
     return calls
 
 
-def sanger(store, mail, institute_obj, case_obj, user_obj, variant_obj, sender, 
-          url_builder=url_for):
-    """Send Sanger email."""
-    variant_link = url_builder('variants.variant', institute_id=institute_obj['_id'],
-                               case_name=case_obj['display_name'],
-                               variant_id=variant_obj['_id'])
-    if 'suspects' in case_obj and variant_obj['_id'] not in case_obj['suspects']:
-        store.pin_variant(institute_obj, case_obj, user_obj, variant_link, variant_obj)
+def variant_verification(store, mail, institute_obj, case_obj, user_obj, variant_obj, sender, variant_url, order, url_builder=url_for):
+    """Sand a verification email and register the verification in the database
+
+        Args:
+            store(scout.adapter.MongoAdapter)
+            mail(scout.server.extensions.mail): an instance of flask_mail.Mail
+            institute_obj(dict): an institute object
+            case_obj(dict): a case object
+            user_obj(dict): a user object
+            variant_obj(dict): a variant object (snv or sv)
+            sender(str): current_app.config['MAIL_USERNAME']
+            variant_url(str): the complete url to the variant (snv or sv), a link that works from outside scout domain.
+            order(str): False == cancel order, True==order verification
+            url_builder(flask.url_for): for testing purposes, otherwise test verification email fails because out of context
+    """
 
     recipients = institute_obj['sanger_recipients']
     if len(recipients) == 0:
         raise MissingSangerRecipientError()
 
+    view_type = None
+    email_subject = None
+    category = variant_obj.get('category')
+    display_name = None
+    breakpoint_1 = str(variant_obj.get('chromosome'))+':'+str(variant_obj.get('position'))
+    breakpoint_2 = None
+    if variant_obj.get('end_chrom'):
+        breakpoint_2 = str(variant_obj.get('end_chrom'))+':'+str(variant_obj.get('end'))
```

Comment:
Use `variant_obj['end_chrom']` . You already check if it exists with `get` above.
I would prefer:
```suggestion
chrom = variant['chromosome'] # We know this exists
end_chrom = variant.get('end_chrom', chromosome)

breakpoint_1 = ':'.join([chromosome, str(variant_obj['position']])
breakpoint_2 = ':'.join([end_chrom, str(variant_obj.get('end')])
.
.
. 
```